MetadataExtractor/Formats/Png/PngChromaticities.cs

```diff
@@ -20,23 +20,16 @@ public PngChromaticities(byte[] bytes)
             if (bytes.Length != 8 * 4)
                 throw new PngProcessingException("Invalid number of bytes");
 
-            var reader = new SequentialByteArrayReader(bytes);
+            var reader = new BufferReader(bytes, isBigEndian: true);
 
-            try
-            {
-                WhitePointX = reader.GetInt32();
-                WhitePointY = reader.GetInt32();
-                RedX = reader.GetInt32();
-                RedY = reader.GetInt32();
-                GreenX = reader.GetInt32();
-                GreenY = reader.GetInt32();
-                BlueX = reader.GetInt32();
-                BlueY = reader.GetInt32();
-            }
-            catch (IOException ex)
-            {
-                throw new PngProcessingException(ex);
-            }
```

Comment:
We validate the length of the byte array (8 *4) above, proving this can't throw.